deploy/staging/database/database-stage.tfvars

```diff
@@ -0,0 +1,12 @@
+// Remote State
+remote_state_bucket = "solo-stage-ecs-fargate-terraform-remote-state"
+remote_state_key    = "STAGE/platform-stage.tfstate"
+region              = "us-gov-west-1"
+project             = "SOLO"
+
+internet_cidr_block = "0.0.0.0/0"
```

Comment:
recommend hardcoding this

Reply:
yes to "region" and "cidr". no to "project". It gets used too often.